### DISCUSSION TITLE
twarc followers

### DISCUSSION THREAD

**Discussion by miguelwon** [DocNow/twarc#618] [Q&A] [answered]
Twitter API followers endpoint returns the list of followers sorted by the moment the user followed the account. 

1. Does `twarc followers` command output respect that order? 
2. Is it possible to get details about each user metadata when retrieving the followers?

**Comment by igorbrigadir** [comment, marked as answer]
1. Yes, twarc returns exactly what the API does, it only modifies the original responses to add some metadata in `__twarc` so if the API response is ordered somehow, the twarc response will be too.
2. By default, all available expansions and fields are requested, so the full user payload is returned. The original responses have all of this inside `includes` portion of the response, and if you want to merge everything and have 1 complete object per line, you can use `twarc2 flatten` command on the responses, and you'll get 1 user per line.
